specifications/android.md

```diff
@@ -49,18 +51,21 @@ The exposed broadcast receiver of the distributor application MUST handle 2 diff
 
 ### org.unifiedpush.android.distributor.REGISTER
 
-The connector send this action to register to push messages. The intent MUST contain 2 String extras:
-* application: with the end user application package name. The distributor MUST be able to handle many connections with a single application.
-* token: with a random token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+The connector send this action to register to push messages. The intent MUST contain 2 extras:
+* application (String): with the end user application package name. The distributor MUST be able to handle many connections with a single application.
+* token (String): with a random token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+
+It MAY be send with the following extra:
+* features (ArrayList<String>): indicate the connector supports a set of optional features. Only "bytesMessage" is currently specified.
```

Comment:
Should this be the FQCN of the feature?

Reply:
The supported features should be listed somewhere else I think